server/venueless/api/auth.py

```diff
@@ -0,0 +1,113 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import authentication, exceptions, permissions
+from rest_framework.authentication import get_authorization_header
+
+from venueless.core.models import Room, World
+from venueless.core.services.world import has_permission
+from venueless.core.utils.jwt import _decode_token
```

Comment:
We should probably rename the method to ``decode_token`` if we're using it in a different module.

Reply:
Yeah, and what about the thing that currently is _decode_token? I was trying to avoid to many sync_to_async and async_to_sync's in one view, as that sounds bad from a performance point of view

Reply:
How about making it a method on `World`? It comes out to the same thing and is more  understandable, I'd think. `world.decode_token()` seems like a good API.